src/main/java/objects/SkillRequirements.java

```diff
@@ -0,0 +1,28 @@
+package objects;
+
+import constants.Skills;
+
+import java.util.EnumMap;
+
+//Wrapper class for skill requirements
+public class SkillRequirements {
+	EnumMap<Skills, Integer> skills;
+
+	public SkillRequirements (){
```

Comment:
This constructor should require the array of requirements, as currently the value of each skill is 1 with no way to set it.